_includes/master/selectors.md

```diff
@@ -2,9 +2,12 @@ A label selector is an expression which either matches or does not match an endp
 
```

Comment:
There are some changes in the v3.0 one that aren't in this file.

namely, "endpoint" -> "a resource"

Reply:
@tmjd just missing one diff in this file still

Reply:
Ok should be updated now. Sorry I copied over the whole table and didn't realize it was something in the text above it.